Model/Product/Handler/Message/SynchronizeProductMessageHandler.php

```diff
@@ -112,6 +120,7 @@ public function __construct(
         string $syliusBaseUrl,
         bool $autoPublish
     ) {
+        $this->logger = $logger;
```

Comment:
if the passt logger is null create a NullLogger: `$this->logger = $logger ?? new NullLogger()`